content/agent/about.md

```diff
@@ -1,57 +1,68 @@
 ---
-title: "About"
+title: About
+draft: false
 weight: 100
 toc: true
-docs: DOCS-000
+nd-docs: DOCS-1091
+nd-content-type:
+- how-to
```

Comment:
```suggestion
nd-content-type: how-to
```

Should be one line: we map content types 1:1 with a particular template or archetype at the moment.

A multi-value field would be applicable for tracking intended/assumed personas.